### PR TITLE
[BX-1057] ‘Del’ text slightly exceeds button width

### PR DESCRIPTION
**Pull request by derHowie**
## What changed (plus any additional context for devs)
Let multicharacter button strings expand the width of the bubble keys

## Screen recordings / screenshots
Screen Shot 2023-10-03 at 7 14 46 AM

## What to test
Make sure button strings have sufficient horizontal padding inside the bubble keys

### CODE REVIEW

**Review by estebanmino — COMMENTED** [review]
i feel like we should only do this when the shortcut has more than one character

for example

image


W is bigger than other letters so in the app menu these shortcut hints look misaligned

wdyt?

**Comment by derHowie** [comment]
> i feel like we should only do this when the shortcut has more than one character
> 
> for example
> 
> image
> W is bigger than other letters so in the app menu these shortcut hints look misaligned
> 
> wdyt?

Good call, fixed.

Screen Shot 2023-10-04 at 12 41 12 AM

**Review by greg-schrammel — APPROVED** [review]

**Comment by estebanmino** [comment]
> > i feel like we should only do this when the shortcut has more than one character
> > for example
> > image
> > W is bigger than other letters so in the app menu these shortcut hints look misaligned
> > wdyt?
> 
> Good call, fixed.
> 
> Screen Shot 2023-10-04 at 12 41 12 AM

hm it doesn't look fixed 

image

File: src/entries/popup/components/ShortcutHint/ShortcutHint.tsx [hunk 1]
```diff
@@ -11,13 +11,14 @@ export const ShortcutHint = ({
 }) => {
   const height = small ? '14px' : '18px';
   const width = small ? '16px' : '18px';
+  const widthKey = hint?.length || 0 > 1 ? 'minWidth' : 'width';
```
Comment (estebanmino) [inline src/entries/popup/components/ShortcutHint/ShortcutHint.tsx:14]:
```suggestion
  const widthKey = (hint?.length || 0) > 1 ? 'minWidth' : 'width';
```

**Review by brunobar79 — APPROVED** [review]
🤝

**Pull request merged**